types/src/shared/feature_flags.ts

```diff
@@ -2,6 +2,7 @@ export const WHITELISTABLE_FEATURES = [
   "usage_data_api",
   "okta_enterprise_connection",
   "labs_transcripts",
+  "labs_transcripts_datasource",
```

Comment:
Do we need another flag for this? Can't we use the existing `labs_transcripts`?

Reply:
labs_transcripts activates the transcripts, for now as this PR is UI only I need to hide the datasource part to current transcripts users, so second FF needed. Will be removed when backend part is shipped